### PR TITLE
Add classes to generate new reports

### PR DESCRIPTION
**Pull request by JRojas999**

### CODE REVIEW

File: src/test/java/TestRunner.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import org.testng.annotations.AfterSuite;
+import salesforce.ReportingUtils;
+
+public class TestRunner extends AbstractTestNGCucumberTests {
+
+    @AfterSuite
+    public void generateReport() {
+        ReportingUtils.generateJVMReport();
+    }
+
```
Comment (SaulCM) [inline src/test/java/TestRunner.java]: Extra spaces should be removed.

File: src/main/java/salesforce/ReportingUtils.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package salesforce;
```
Comment (SaulCM) [inline src/main/java/salesforce/ReportingUtils.java]: Probably missing the copyright.

File: src/main/java/salesforce/ReportingUtils.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import net.masterthought.cucumber.presentation.PresentationMode;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class ReportingUtils {
+
+    public static void generateJVMReport() {
+
```
Comment (SaulCM) [inline src/main/java/salesforce/ReportingUtils.java]: Empty line.

**Review by SaulCM — APPROVED** [review]

**Review by danielaSantaCruz — APPROVED** [review]

File: src/test/java/TestRunner.java (hunk 1 above)
Comment (JRojas999) [reply src/test/java/TestRunner.java]: Good, it will

File: src/main/java/salesforce/ReportingUtils.java (hunk 3 above)
Comment (JRojas999) [reply src/main/java/salesforce/ReportingUtils.java]: I reviewed checkstyle mistakes.

**Pull request merged**